### PR TITLE
Refactor tests, and unify page_ordinal -> ordinal in urlconf

### PR DESCRIPTION
**Pull request by dzhuang**
`page_ordinal` and `ordinal` are unified into `ordinal` in `URLCONF` and views.

### CODE REVIEW

File: course/auth.py [hunk 1]
```diff
@@ -270,6 +270,24 @@ def stop_impersonating(request):
         raise PermissionDenied()
 
     if not hasattr(request, "relate_impersonate_original_user"):
+        # prevent user without pperm to stop_impersonating
+        my_participations = Participation.objects.filter(
+            user=request.user,
+            status=participation_status.active)
```
Comment (inducer) [inline course/auth.py]: So `request.user` is the impersonated user, which means that getting out of impersonating someone now would require a logout?

File: course/grading.py [hunk 2]
```diff
@@ -88,7 +88,7 @@ def get_prev_visit_grades(
 
 
 @course_view
-def get_prev_grades_dropdown_content(pctx, flow_session_id, page_ordinal):
+def get_prev_grades_dropdown_content(pctx, flow_session_id, ordinal):
```
Comment (inducer) [inline course/grading.py]: I think I would've preferred standardizing the other way. `ordinal` is less descriptive than `page_ordinal`.

File: course/auth.py (hunk 1 above)
Comment (dzhuang) [reply course/auth.py]: No, it was preventing who didn't currently impersonating and who don't have impersonating pperm to do `stop_impersonating`. You can see from the testcase [here](https://github.com/inducer/relate/pull/388/files#diff-b4c9abe346f4b3686dd3e1025a50aaa7R116)

File: course/grading.py (hunk 2 above)
Comment (dzhuang) [reply course/grading.py]: Ok. Is it necessary to standardize that name in model?  (needs migrations).

Comment (dzhuang) [reply course/grading.py]: Updated, wth `FlowPageData.ordinal` renamed to `page_ordinal`.

Comment (inducer) [reply course/grading.py]: :/

File: course/auth.py (hunk 1 above)
Comment (dzhuang) [reply course/auth.py]: Without this, student user can view the `impersonate` page. This patch redirects them to 403.

Comment (dzhuang) [reply course/auth.py]: I know that requires extra db hits when stop impersonating, but I just want them to know there are permissions.

**Comment by inducer** [comment]
Thanks!

File: course/auth.py (hunk 1 above)
Comment (dzhuang) [reply course/auth.py]: > This patch redirects them to 403.

The tests is (here)[https://github.com/inducer/relate/pull/388/files#diff-b4c9abe346f4b3686dd3e1025a50aaa7R81]. Without this, the `status_code` is 200(302).

Comment (inducer) [reply course/auth.py]: Sure. It doesn't hurt. Thanks for working on this!

**Pull request merged**